### DISCUSSION TITLE
v0.6.2

### DISCUSSION THREAD

**Discussion by U-C-S** [U-C-S/Hurl#45] [Announcements]
## Feature Updates
- Support `CustomIconPath` (#43)
- You can now use `%URL%` in LaunchArgs as a param that's replaced by URL at runtime (#43)
- Make some fields in `UserSettings.json` as Optional

## Fixes
- Fix crashing when launching after a fresh install (#42)
- Fix crash when upgrading from `v0.6`  (This is due to non-optional fields in `UserSettings.json`)

## Credits
Thanks to @hanjo , @cwichel , @grentninja for feedback and suggestions

**Full Changelog**: https://github.com/U-C-S/Hurl/compare/v0.6.1...v0.6.2

<hr /><em>This discussion was created from the release <a href='https://github.com/U-C-S/Hurl/releases/tag/v0.6.2'>v0.6.2</a>.</em>

**Comment by grentninja** [comment]
Nice! Will download this new version later :)

Glad to see to see an update from Hurl, keep it up!

**Reply by hanjo** [reply]
Thank you, the new features are great! 😊
